SPECS/kernel-alt.spec

```diff
@@ -23,7 +23,7 @@
 Name: kernel-alt
 License: GPLv2
 Version: 4.19.142
-Release: 1%{?dist}
+Release: 2%{?dist}
```

Comment:
Since Release 1 never got built, due to the missing dependency, no need to increment it.

Same goes for the changelog, you just need to add a line to the 4.19.142-1 changelog entry (and update the date if you want).

Reply:
Thanks. Updated it.